syntax = "proto3";


  message RFD {
    string RFWID =1;
    int32 LastBatchID =2;
    repeated double dataRequested =3;
    double dataAnalytics =4;
  }
message RFW {
    string RFWID =1;
    string benchmarkType =2;
    string workloadMetric =3;
    int32 batchUnit =4;
    int32 batchID = 5;
    int32 batchSize =6;
    string dataType =7;
    string dataAnalytics =8;
  }